classes/webservice/ApiNode.php

```diff
@@ -0,0 +1,237 @@
+<?php
+
+/**
+ * Description of ApiNode
+ *
+ * @author Matej Kminek <matej.kminek@attendees.eu>, 28. 1. 2021
+ */
```

Comment:
Please include the PrestaShop license in your files.

Reply:
Included in new files ApiNode & SuperXMLElement